packages/grafana-data/src/types/scopes.ts

```diff
@@ -1,20 +1,31 @@
-export interface ScopeDashboard {
-  uid: string;
-  title: string;
-  url: string;
+export interface ScopeDashboardBindingSpec {
+  dashboard: string;
+  scope: string;
 }
 
-export interface ScopeFilter {
+export interface ScopeSpecFilter {
   key: string;
   value: string;
   operator: string;
 }
 
-export interface Scope {
-  uid: string;
+export interface ScopeSpec {
   title: string;
   type: string;
   description: string;
   category: string;
+  filters: ScopeSpecFilter[];
+}
+
+export interface ScopeDashboard {
+  uid: string;
+  title: string;
+  url: string;
+}
+
+export type ScopeFilter = ScopeSpecFilter;
+
+export interface Scope extends ScopeSpec {
```

Comment:
?? is this a "public" replacement for what should eventually be:

`Resource<ScopeSpec, 'Scope'>`

Reply:
Yeah. To be honest I thought that internal models should not be nested objects with spec and meta but rather plain objects. I can change it tho' as on the long run we might need additional props from meta and they'll just be there as opposed to adding them manually.

Reply:
right -- the default object has a lot more then you would add for a simple flavor.  In playlist, we convert between the k8s form and the "simple" object -- long term I suspect that conversion adds more confusion than is useful and becomes a mess when you want to see things like who edited what when

Reply:
Alrighty then. I'm moving back to passing the `Resource` as is instead of manipulating it to plain objects. The only problem with it is that we haven't exposed the API server types yet.

I can approach this in two ways:
- hardcode the interfaces for the time being with a TODO message attached to switch to `Resource` once we expose the types
- expose the API server types now and think carefully if we need to change them

WDYT?